services/fncsgossbridge/service/fncs_goss_bridge.py

```diff
@@ -262,6 +263,14 @@ def run_simulation(self,run_realtime):
     def on_message(self, headers, msg):
         message = {}
         try:
+            headers_dict = yaml.safe_load(str(headers))
+            destination = headers_dict['destination']
+            message_id = headers_dict['message-id']
+            if str(destination).startswith('/temp-queue'):
+                return
+            if str(message_id) in self.message_id_list:
+                return
+            self.message_id_list.append(str(message_id))
```

Comment:
Why are these changes here ? Update from upstream develop and update pull request.